docker-compose.yml

```diff
@@ -199,3 +199,32 @@ services:
       SOURCE_FILES_DATA_GRAPH: "http://mu.semte.ch/graphs/original-physical-files-data"
     volumes:
       - ./data/files:/share
+  ################################################################################
+  # OP PUBLIC CONSUMER
+  ################################################################################
+  op-public-consumer:
+    image: lblod/delta-consumer:0.0.27
```

Comment:
You can bump directly to v0.1.0, shouldn't change the logic, it "just" handles language tags better :)